File: fizzbuzz.js
function fizzbuzz(item) {
    switch (true) {
        case item % 15 === 0:
            console.log('fizzbuzz');
            break;
        case item % 3 === 0:
            console.log('fizz');
            break;
        case item % 5 === 0:
            console.log('buzz');
            break;
        default:
            console.log(item);
            break;
    }
};

File: fizzbuzz.rb
class Fizzbuzz
  def self.run
    (1..100).map do |number|
    case
      when number % 3 == 0
        'fizz'
      when number % 5 == 0
        'buzz'
      when number % 15 == 0
        'fizzbuzz'
      else
        number
    end
  end
end
end
Fizzbuzz.run